pkg/broker/ingress/handler.go

```diff
@@ -99,7 +99,7 @@ func NewHandler(ctx context.Context, httpReceiver HttpMessageReceiver, decouple
 
 // Start blocks to receive events over HTTP.
 func (h *Handler) Start(ctx context.Context) error {
-	return h.httpReceiver.StartListen(ctx, h)
+	return h.httpReceiver.StartListen(ctx, kncloudevents.CreateHandler(h))
```

Comment:
I think StartListen might also wrap the handler?

Reply:
Yes it did, thanks!